step/src/main/webapp/profile.jsp

```diff
@@ -111,7 +111,19 @@ limitations under the License.
                     <div class="item-info"> <%= (String) entity.getProperty("q5")%> </div>
                 </div> 
             </div>
-            <h3> Your Matches </h3>
+            <h3> <a href="chat.jsp"> Your Matches </a> </h3>
```

Comment:
Can you change the hyperlink to chat.jsp to be somewhere else? Maybe like a "Click to chat to your matches" button within the matches-container div? It's currently very unintuitive/hidden and I think most user won't realize that a chat even exists.

Reply:
I agree with that! Currently, I leave the hyperlink there as a temporary way to get access to chat.jsp file. But in the future, the users can click on your match's profile picture and it will direct the users to the chat box with their specific chosen match.